src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromDevider = fromFormat[3];
```

Comment:
There is a typo in the variable name 'fromDevider'. The correct spelling should be 'fromDivider'.